API_register/models.py

```diff
@@ -56,6 +21,13 @@ def get_classes(self):
         return self.classes_set.all()
 
 
+class AnnouncementManager(models.Manager):
```

Comment:
What about delete method? What should happen when you make `Announcements.objects.get(id=1).delete()` ?